### PR TITLE
Chore/add images

### LINKED ISSUE #4: Upload SQL files (zip codes) to PostgreSQL using Heroku

### LINKED ISSUE #9: Research PostgreSQL

### LINKED ISSUE #11: Deploy practice scenic routes API to Heroku

### LINKED ISSUE #12: Input tree overview data into CSV

### LINKED ISSUE #14: Add images to assets folder

### LINKED ISSUE #16: Add tree-specific folders to assets folder

### PR DESCRIPTION
**Pull request by BrandonAinsworth**
What does this PR do?
- Uploaded zip_codes SQL file to PostgreSQL database
- Deployed practice scenic-routes API to Heroku
- Added tree overview data into CSV
- Added images into assets folder

Where should your reviewer start?
- Line 3 in .gitignore
- Line 145 in old_ZipCodesTable.sql

What (if any) features are you implementing?
- N/A

What (if anything) did you refactor?
- N/A

Were there any issues that arose?
- N/A

Is there anything that you need from your teammate?
- N/A

Any other comments, questions, or concerns?
- Resolves #4 
- Resolves #9
- Resolves #11 
- Resolves #12
- Resolves #14 
- Resolves #16

### CODE REVIEW

File: old_ZipCodesTable.sql [hunk 1]
```diff
@@ -142,6 +142,7 @@ INSERT INTO zip_codes (id, sid, default_state, city, default_type, zip_code, har
 (5155, 7, 'CO', 'DENVER', 'STANDARD', '80294', '5b'),
 (5156, 7, 'CO', 'DENVER', 'STANDARD', '80299', '5b')
 (5166, 7, 'CO', 'DURANGO', 'STANDARD', '81301', '6a'),
+-- 5166 was causing an error on terminal
```
Comment (irmakerdem) [inline old_ZipCodesTable.sql:145]: This was very weird, still don't know why it gave an error 🤔

**Review by rachel-bock — COMMENTED** [review]
Lots of work here. 😫  Glad you were able to get the images of the trees 🌴 🎄 🌳 🌲  and learn about the new webp image file format.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 .DS_Store
 node_modules
+.env
```
Comment (irmakerdem) [inline .gitignore:3]: Nice job keeping our app safe by not allowing the password to be public! 🤫

**Review by Baskanbetul — APPROVED** [review]
Thank you for adding assets .

File: old_ZipCodesTable.sql (hunk 1 above)
Comment (rachel-bock) [reply old_ZipCodesTable.sql:145]: There's no comma at the end of line 143.  It was something we were able to fix when we imported the data into the database.  😄

**Review by irmakerdem — APPROVED** [review]
Looking good! Excited to see how the real tree data will get deployed! 🌲 🍃

**Pull request merged**